### ISSUE TITLE
Error in checkout button in the store widget

### ISSUE THREAD

**Issue by FabrizzioGM** [woothemes/woocommerce#31892]
Once the order is ready, customers who press "Proceed to check out" in the "cart" page customers get redirected to the home page. And neither can I modify the direction of that button anywhere.

**Comment by jonathansadowski** [comment]
Hi @FabrizzioGM,

Thank you for submitting the issue. However, you didn’t fill out the details of the bug report template that we ask for. Without these details, we can’t fully evaluate this issue. Please fill out [this form](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=&template=1-bug-report.yml) and create a new issue so that we can further evaluate. I am closing this issue.
